cli/pom.xml

```diff
@@ -6,7 +6,7 @@
 	<parent>
 		<groupId>life.qbic</groupId>
 		<artifactId>parent-pom</artifactId>
-		<version>2.3.0-SNAPSHOT</version>
+		<version>3.0.0</version>
```

Comment:
why not version 2.4.0 as mentioned in the release branch? backwards compatibility?

Reply:
it is not backwards compatible, it breaks all implementations. The referencing apps need to declare the dependencies to i.e. the data-model-lib themselves in the future. Sorry, the branch name was misleading

Reply:
i see, makes sense then